src/plots/cartesian/axis_ids.js

```diff
@@ -65,7 +64,7 @@ function listNames(gd, axLetter, only2d) {
     var names = filterAxis(fullLayout, '');
     if(only2d) return names;
 
-    var sceneIds3D = Plots.getSubplotIds(fullLayout, 'gl3d') || [];
```

Comment:
removed one circular dep here (only 16 to go 🙄 ) by having `_subplots` already calculated. I removed `Plots.getSubplotIds` entirely since it had become a 1-liner just looking up entries in `fullLayout._subplots`.